src/wp-includes/cron.php

```diff
@@ -972,30 +972,26 @@ function spawn_cron( $gmt_time = 0 ) {
 }
 
 /**
- * Registers _wp_cron() to run on the {@see 'wp_loaded'} action.
+ * Registers _wp_cron() to run on the {@see 'shutdown'} action.
  *
- * If the {@see 'wp_loaded'} action has already fired, this function calls
- * _wp_cron() directly.
- *
- * Warning: This function may return Boolean FALSE, but may also return a non-Boolean
- * value which evaluates to FALSE. For information about casting to booleans see the
- * {@link https://www.php.net/manual/en/language.types.boolean.php PHP documentation}. Use
- * the `===` operator for testing the return value of this function.
+ * The {@see spawn_cron()} function attempts to make a non-blocking loopback request to `wp-cron.php` (when alternative
+ * cron is not being used). However, the {@see wp_remote_post()} function does not always respect the `timeout` and
+ * `blocking` parameters. A timeout of `0.01` may end up taking 1 second. When this runs at the {@see 'wp_loaded'}
+ * action, it increases the Time To First Byte (TTFB) since the HTML cannot be sent while waiting for the cron request
+ * to initiate. Moving the spawning of cron to the {@see 'shutdown'} allows for the server to flush the HTML document to
+ * the browser while waiting for the request.
```

Comment:
Function names don't need the see annotation as they're parsed by developer.wp.org without them. 

```suggestion
 * The spawn_cron() function attempts to make a non-blocking loopback request to `wp-cron.php` (when alternative
 * cron is not being used). However, the wp_remote_post() function does not always respect the `timeout` and
 * `blocking` parameters. A timeout of `0.01` may end up taking 1 second. When this runs at the {@see 'wp_loaded'}
 * action, it increases the Time To First Byte (TTFB) since the HTML cannot be sent while waiting for the cron request
 * to initiate. Moving the spawning of cron to the {@see 'shutdown'} hook allows for the server to flush the HTML document to
 * the browser while waiting for the request.
```